lib/Wikia/src/Service/User/Permissions/PermissionsServiceImpl.php

```diff
@@ -351,8 +366,7 @@ public function removeFromGroup( \User $performer, \User $userToChange, $groups
 				}
 			}
 		} finally {
-			$this->invalidateCache( $userToChange );
-			$userToChange->invalidateCache();
+			$userToChange->invalidateCache(); // it calls $this->invalidateCache( $userToChange );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)